api/src/test/java/uk/gov/hmcts/payment/api/controllers/FeesControllerTest.java

```diff
@@ -170,7 +170,7 @@ public void deleteFeesCreatedUsingRemissionTest() throws Exception {
     @Test
     public void deleteNoFeesExistingTest() throws Exception {
 
-        Integer feeId = 12;
+        Integer feeId = 99999;
```

Comment:
Out of curiosity why this change?

Reply:
also came from Dave's PR https://github.com/hmcts/ccpay-payment-app/pull/1696/files#diff-9782b254b76669fd72594855a949e988d483dab61fa1e79f7b31a6412a996d84R173